geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -44,10 +45,24 @@ app.module.directive('appLayermanager', app.layermanagerDirective);
 
 /**
  * @constructor
+ * @param {angular.Scope} $scope
+ * @param {ngeo.SyncArrays} ngeoSyncArrays
  * @export
  */
-app.LayermanagerController = function() {
+app.LayermanagerController = function($scope, ngeoSyncArrays) {
   this['uid'] = goog.getUid(this);
+  ngeoSyncArrays(this['map'].getLayers().getArray(),
+      this['layers'], true, $scope,
+      goog.bind(function(layer) {
+        return goog.isDefAndNotNull(layer['values_']['metadata']);
```

Comment:
The use of `ngeoSyncArrays` to synchronize the array of layers in the map with the array of selected layers should be in the main controller (in place of what currently is in the `manageSelectedLayers_` method) . Indeed, the layer manager should not know about background layers, so it should not know how to filter out background layers.